documentation/mDNS_Support.md

```diff
@@ -20,17 +20,18 @@ Our sample file `physical-web-url.service` looks like this:
 <!DOCTYPE service-group SYSTEM "avahi-service.dtd">
 
 <service-group>
-  <name replace-wildcards="yes">http://www.mycompany.com/xyz.html</name>
+  <name replace-wildcards="yes">Physical Web on %h</name>
   <service>
-    <host-name>www.mycompany.com</host-name>
-    <type>_http._tcp</type>
+    <type>_urlbookmark._tcp</type>
```

Comment:
Oh, hm, I see...yea, I guess we should just have a different suggested name then and possibly leave it as is.  Although...the urlbookmark service type seems to be directly targeted at what we are doing.  I'll have to look around a bit more.